src_features/signMessageEIP712/commands_712.c

```diff
@@ -38,6 +38,7 @@ void handle_eip712_return_code(bool success) {
 
     if (!success) {
         eip712_context_deinit();
+        ui_idle();
```

Comment:
Not on stax ?

Reply:
This code is common to all devices so it applies to Stax as well. :+1: